apps/moderation/serializers.py

```diff
@@ -16,3 +17,69 @@ class Meta(model_serializer.Meta):
             fields = model_serializer.Meta.fields + ['fields']
 
     return ModeratedModelSerializer
+
+
+class PythonSerializer(Serializer):
```

Comment:
@alashow, can you add a docstring explaining how this extends Django's `Serializer` (what it does differently)?

Reply:
I added a docstring. However, I don't think this an ideal solution, we should report the bug to upstream / django.

Reply:
Gotcha. Thanks!